testbed/compositions/graphsync.toml

```diff
@@ -16,23 +16,20 @@ instances = { count = 2 }
 
 [groups.build]
 
-
-
-
 [groups.run]
 [groups.run.test_params]
-input_data = "files"
-data_dir = "../extra/test-datasets"
+input_data = "random"
+file_size = "2000000000"
 run_timeout_secs = "3000"
 timeout_secs = "12000"
-run_count = "4"
+run_count = "5"
 leech_count= "1"
 passive_count = "0"
 max_connection_rate = "100"
 # input_data = "random"
 # file_size = "10000000,30000000,50000000"
-latency_ms= "100,10"
-bandwidth_mb= "100,50"
+latency_ms= "0"
```

Comment:
Small nitpick.
I guess you are removing latency and bandwidth because you are interested in the performance of graphsync within the peer, right? If this is not the case and you want to test the protocol's performance we should add some of this. 

I would maybe keep latency and bandwidth to the standard 100, 100 or something like that (even if you change it locally for your tests) so we don't encourage zero latency/bandwidth tests.

Reply:
@adlrocha I agree. I've reverted the changes in all the composition files. Will explicitly name the files as `no_latency...` to make sure the reader knows the composition does NOT use latency and is ONLY for benchmarking purposes.

Reply:
Great idea!